scripts/tests/chiptest/linux.py

```diff
@@ -88,23 +95,33 @@ def CreateNamespacesForAppTest():
         "ip netns exec app ip link set dev lo up",
         "ip link set dev eth-app-switch up",
 
-        "ip netns exec tool ip addr add 10.10.10.2/24 dev eth-tool",
+        "ip netns exec tool ip addr add 10.10.12.2/24 dev eth-tool",
         "ip netns exec tool ip link set dev eth-tool up",
         "ip netns exec tool ip link set dev lo up",
         "ip link set dev eth-tool-switch up",
 
-        # Force IPv6 to use ULAs that we control
-        "ip netns exec tool ip -6 addr flush eth-tool",
-        "ip netns exec app ip -6 addr flush eth-app",
-        "ip netns exec tool ip -6 a add fd00:0:1:1::2/64 dev eth-tool",
-        "ip netns exec app ip -6 a add fd00:0:1:1::3/64 dev eth-app",
-
-        # create link between virtual host 'tool' and the test runner
         "ip addr add 10.10.10.5/24 dev eth-ci",
+        "ip addr add 10.10.12.5/24 dev eth-ci",
         "ip link set dev eth-ci up",
         "ip link set dev eth-ci-switch up",
     ]
 
+    if not ble_wifi:
+        COMMANDS += [
+            "ip link add eth-app-direct type veth peer name eth-tool-direct",
```

Comment:
Please have @andy31415 review these bits; I am not very familiar with the netns stuff.